README.md

```diff
@@ -63,13 +59,37 @@ pip install -r requirements.txt
 
 Alternatively you can install it using your Linux distro's package manager
 
+### Authentication
+
+First, you neeed to obtain data about account `access_token`, `refresh_token` and `user_id`
+
+(for Heroic these can be found in `$HOME/.config/heroic/gog_store/config.json`)
```

Comment:
Could you edit/remove that part while you are on it? The config.json is no longer used in favor of auth.json

Reply:
Okay, just corrected that without changing the readme structure